lib/sedna/README.md

```diff
@@ -1,2 +1,224 @@
+        metrics = [keras.metrics.categorical_accuracy]
+        optimizer = keras.optimizers.Adam(learning_rate=0.1)
+        model.compile(loss=loss, optimizer=optimizer, metrics=metrics)
+        return model    
+     ```
+
+    ```python
+    # XGBOOST
+    
+    import os
+    import xgboost
+ 
+    os.environ['BACKEND_TYPE'] = 'SKLEARN'
+ 
+    XGBEstimator = xgboost.XGBClassifier(
+            learning_rate=0.1,
+            n_estimators=600,
+            max_depth=2,
+            min_child_weight=1,
+            gamma=0,
+            subsample=0.8,
+            colsample_bytree=0.8,
+            objective="multi:softmax",
+            num_class=3,
+            nthread=4,
+            seed=27
+     )
+ 
+    ```
+    
+    ```python
+ 
+    # Customize
+ 
+    class Estimator:
+
+        def __init__(self, **kwargs):
+            ...
+            
+        def load(self, model_url=""):
+            ...
+            
+        def save(self, model_path=None):
+            ...
+
+        def predict(self, data, **kwargs):
+            ...
+            
+        def evaluate(self, valid_data, **kwargs):
+            ...
+            
+        def train(self, train_data, valid_data=None, **kwargs):
+            ...
+ 
+	```
+    > **Notes**:  Estimator is a high-level API that greatly simplifies machine learning programming. Estimators encapsulate training, evaluation, prediction, and exporting for your model. 
+
+
+3. Initialize a Incremental Learning Job:
+
+    ```python
+   
+    # get hard exmaple mining algorithm from config
+    hard_example_mining = IncrementalLearning.get_hem_algorithm_from_config(
+        threshold_img=0.9
+    )
+   
+    # create Incremental Learning infernece instance
+    il_job = IncrementalLearning(
+        estimator=Estimator,
+        hard_example_mining=hard_example_mining,
+    )
+   
+    ```
+
+	where:
+   
+	- `IncrementalLearning` is the Cloud-edge job you want to access.
+	- `Estimator` is the base model for your ML job.
+	- `hard_example_mining` is the parameters of incremental learning job.
+    
+    Inference
+    ---------
+    
+	> **Note:** The `job parameters` of each feature are different.
+
+4. Running Job - training / inference / evaluation.
+
+	```python
+	results, final_res, is_hard_example = il_job.inference(
+            img_rgb, 
+            post_process=deal_infer_rsl, 
+            input_shape=input_shape
+    )
+ 
+	```
+ 	
+	where:
+
+	- `img_rgb` is the sample used to inference
+    - `deal_infer_rsl` is a function used to process result after model predict
+    - `input_shape` is the parameters of `Estimator` in inference
+    - `results` is the result predicted by model
+    - `final_res` is the result after process by `deal_infer_rsl`
+    - `is_hard_example` tells if the sample is hard sample or not
+
+## Customize algorithm
+
+Sedna provide a class called `class_factory.py` in `common` package, only a few lines of changes are required to become a module of sedna.
```

Comment:
1、provide -> provides
2、, in which only a few lines of changes...

Reply:
check